packages/mini-browser/src/browser/mini-browser-content.ts

```diff
@@ -511,8 +515,16 @@ export class MiniBrowserContent extends BaseWidget {
         return element;
     }
 
-    protected mapLocation(location: string): Promise<string> {
-        return this.locationMapper.map(location);
+    protected async mapLocation(location: string): Promise<string> {
```

Comment:
I think we should revert it. What you really want it is to wrap the entire mini browser inside another iframe which is running on another origin.

Reply:
The mini-browser is already served from a different origin, I just messed up the logic to load resources, will fix.